apps/tasks/migrations/0001_initial.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 3.2.9 on 2021-11-14 10:31
+
+from django.db import migrations, models
+import django.db.models.deletion
+import enumchoicefield.fields
+import tasks.models
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Task',
+            fields=[
+                ('id', models.UUIDField(default=uuid.uuid4, editable=False,
+                 primary_key=True, serialize=False, unique=True)),
```

Comment:
primary key by definition is unique so I think you can remove it
```suggestion
                 primary_key=True, serialize=False)),
```